utils/instant-observability-categories.json

```diff
@@ -0,0 +1,72 @@
+[
```

Comment:
I'm not sure why this file is here (I may not have gotten to that part of the PR yet), quickstarts are associated with their categories via the keywords they use, so I can't see any obvious use for this. Especially since when we move over to SSR, we will be getting all of these from the API, so I could see this getting out of date very quickly.

Reply:
I think I brought this over initially to get the categories from user-provided keywords.

If there's another source of the mapping from keywords to categories that's centralized, that would be great to use. I had the same "falling out of sync" concern when I copied this JSON from here: https://github.com/newrelic/developer-website/blob/develop/src/data/instant-observability-categories.json

Reply:
Keeping this for backwards compatibility!